tqdm/rich.py

```diff
@@ -119,6 +119,11 @@ def __init__(self, *args, **kwargs):
     def close(self):
         if self.disable:
             return
+
+        # Add call display to print 100%
+        # https://github.com/tqdm/tqdm/issues/1306#issuecomment-1322762835
+        self.display()
```

Comment:
```suggestion
        if self.leave:
            self.display()
```

should it be with self.leave check like gui.py?
- https://github.com/tqdm/tqdm/blob/6791e8c5b3d6c30bdd2060c346996bfb5a6f10d1/tqdm/gui.py#L104-L105